ts/model_service_worker.py

```diff
@@ -28,8 +29,16 @@ class TorchModelServiceWorker(object):
     Backend worker to handle Model Server's python service code
     """
 
-    def __init__(self, s_type=None, s_name=None, host_addr=None, port_num=None):
+    def __init__(
+        self,
+        s_type=None,
+        s_name=None,
+        host_addr=None,
+        port_num=None,
+        metrics_config=None,
```

Comment:
Would it make sense to pass the Cache directly so the worker would not have to care about the impl detail that this is a yaml configured cache? Worker could only expect MetricCacheAbstract (or we call it MetricCache, can still be abstract)

Reply:
The worker is started from the java frontend, this would require additional serializing/deserializing. We can take it up as a follow up